smoove.wdl

```diff
@@ -0,0 +1,68 @@
+version 1.0
+
+# MIT License
+#
+# Copyright (c) 2020 Leiden University Medical Center
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+task CallSV {
+    input {
+        File bamFile
+        File bamIndex
+        File referenceFasta
+        File referenceFastaFai
+        String sample
+        String outputDir = "./smoove"
+
+        String memory = "15G"
+        String dockerImage = "quay.io/biocontainers/smoove:0.2.5--0"
+    }
+
+    command {
+        set -e
+        mkdir -p "$(dirname ~{outputDir})"
```

Comment:
Will the tool complain if the output directory already exists? Otherwise the following might be more appropriate, since outputDir is supposed to be a directory anyway:
```suggestion
        mkdir -p ~{outputDir}
```